weaver/store/base.py

```diff
@@ -145,6 +146,7 @@ def find_jobs(self,
                   sort=None,                # type: Optional[str]
                   page=0,                   # type: int
                   limit=10,                 # type: int
+                  datetime=None,            # type: Optional[DatetimeIntervalType]
```

Comment:
same incorrect typing

Reply:
I created a [`Datetime type`](https://github.com/crim-ca/weaver/pull/256/files#diff-db2ae0d10aa9d2524d128447d16494063e73c6b2e034c7565885264a1d62a050R44) because it's not really a datatime, it's a string